frontend/src/tests/components/common/RenderCheckbox.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import RenderCheckbox from '@/components/common/RenderCheckbox';
+
+let props = {};
+
+const setupProps = () => {
+  props = {
+    id: 1,
+    input: '',
+    label: '',
+    meta: {
+      touched: false,
+      error: false,
+    },
+  };
+}
+
+beforeEach(() => {
+  setupProps();
+});
+
+describe('RenderCheckbox', () => {
+
```

Comment:
The other spec file doesn't have newline padding around the `it` blocks. Which would you prefer? If we are using the Airbnb eslint config, they disallow newline padding.

https://github.com/airbnb/javascript#whitespace--padded-blocks

Reply:
no newline padding, updated spec files that have it.